storage-service/src/test/java/com/stnetix/ariaddna/persistence/services/AccessTokenServiceImplTest.java

```diff
@@ -0,0 +1,33 @@
+package com.stnetix.ariaddna.persistence.services;
+
+import com.stnetix.ariaddna.commonutils.DTO.AccessTokenDTO;
+import com.stnetix.ariaddna.persistence.utils.AppConfiguration;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+
+import java.util.UUID;
+
+public class AccessTokenServiceImplTest {
+    private ConfigurableApplicationContext context;
+    private AccessTokenServiceImpl accessTokenService;
+
+    @Before
+    public void before() {
+        context = SpringApplication.run(AppConfiguration.class);
```

Comment:
Context will be load in test like as usually in application. Your test check persisting data in DB or not. In your code test data will be stored in database after test is done. It is bad practice.
To change it, you need to mark test class as @Transactional, it will rollback the transaction at the end of each test method by default. It will be work if you mark test class with annotations:
@RunWith(SpringJUnit4ClassRunner.class)
@SpringBootTest(classes = AppConfiguration.class)

And finnally you can teke your beas as @Autowired annotation.